tests/fixtures.py

```diff
@@ -201,6 +201,12 @@ def elasticapm_client(request):
     assert not client._transport.validation_errors
 
 
+@pytest.fixture()
+def elasticapm_transaction(elasticapm_client):
+    transaction = elasticapm_client.begin_transaction("test")
+    yield transaction
```

Comment:
Do we need this fixture? It's only saving you one line in the test you wrote, since you end the transaction and inspect the spans in the end anyway. Actually, since you have to do `get_client()`, it's saving you no lines...

Reply:
the idea is that this way the transaction starts _before_ the `postgres_connection` fixture runs, which enables us to capture the `connect` span.

I can add a comment to explain why it is useful.

Reply:
Ohhhh, thank you. Yes, please add a comment.